Carta.py

```diff
@@ -480,8 +599,44 @@ def pressDoneButton(self, event, mouse):
             (self.phase == Phase.YOUR_SET_UP) and \
             (self.numYourFramesOccupied >= self.numYourGrabCardInPlay)):
             self.phase = Phase.GRABBING
+            self.dialogs.append("Grabbing Phase Starts")
             self.displayReadCardStartTime = self.getTime_ms()
 
+        if ((self.doneButton.collidepoint(event.pos)) and \
+            (self.phase == Phase.YOUR_TRANSFER)):
+            if (self.numCardsToTransfer != 0):
+                self.dialogs.append("Still have " + str(self.numCardsToTransfer) + " cards")
+            elif (self.numCardsToTransfer == 0):
+                selectedTransferCards = []
+                for card in self.grabbingCardsInPlay:
+                    if (card.getColor() is self.colors.yellow):
+                        card.setStatus(GrabCardStatus.OPPONENT)
+                        card.setColor(self.colors.white)
+                        self.occupied[tuple(card.getFrame())] = False
+                        selectedTransferCards.append(card)
+                for transferCard in selectedTransferCards:  # maximum of two
+                    frameIndexList = [k for k in range(len(self.opponentFrames))]
+                    random.seed(time.time())
+                    sampledFrames = random.sample(frameIndexList, self.numOppoGrabCardInPlay)
+                    for i in range(self.numOppoGrabCardInPlay):
+                        frame = self.opponentFrames[sampledFrames[i]]
+                        pygameRect = pygame.rect.Rect(frame[0][0], frame[0][1], self.GUIParameters.cardWidth,
+                                                      self.GUIParameters.cardHeight)
+                        if tuple(frame) not in self.occupied:
+                            transferCard.setRect(pygameRect)
+                            transferCard.setFrame(frame)
+                            self.occupied[tuple(frame)] = True
+                            break
+                        elif tuple(frame) in self.occupied:
+                            if not self.occupied[tuple(frame)]:
+                                transferCard.setRect(pygameRect)
+                                transferCard.setFrame(frame)
+                                self.occupied[tuple(frame)] = True
+                                break
+
+                self.phase = Phase.RESET
+                self.dialogs.append("Reset Phase Starts")
```

Comment:
In debug mode, in grabbing phase, highlight the correct grab card using a different color (not yellow or white or green).
Easier to debug.

When that correct card got removed, set color back to white.